app/views/devise/passwords/new.html.erb

```diff
@@ -0,0 +1,16 @@
+<h2>Forgot your password?</h2>
+
+<%= form_for(resource, as: resource_name, url: password_path(resource_name), html: { method: :post }) do |f| %>
+  <%= render "devise/shared/error_messages", resource: resource %>
+
+  <div class="field">
+    <%= f.label :email %><br />
+    <%= f.email_field :email, autofocus: true, autocomplete: "email" %>
+  </div>
+
+  <div class="actions">
+    <%= f.submit "Send me reset password instructions" %>
```

Comment:
- [ ] **[OPTIONAL]** Password reset works just fine. However, after clicking the button to send me password reset instructions the page navigates to this. I would recommend you fix that.

  You can do that by refactoring this line to this:
  ```
  <%= f.submit "Send me reset password instructions", data: { turbo: false } %>
  ```

  ![fjff](https://user-images.githubusercontent.com/36542986/207610618-2ffd84ef-a893-419b-b199-4fc5998727f4.jpg)